code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1190,3 +1190,47 @@
   */
 /mob/living/carbon/proc/get_biological_state()
 	return BIO_FLESH_BONE
+
+/// Returns maximum number of active chip skills for this mob
+/mob/living/carbon/proc/get_max_skillchip_slot_count()
+	. = 2
+	if(dna && dna.check_mutation(BIOTECHCOMPAT))
```

Comment:
```suggestion
	if(dna?.check_mutation(BIOTECHCOMPAT))
```

Reply:
Couldn't you turn this into a carbon variable that gets incremented and decremented on mutation gain and loss?
That would be much easier to turn this into an event we can respond to in the future.